package.json

```diff
@@ -1,17 +1,16 @@
 {
 	"scripts": {
 		"lint": "xo",
-		"build": "parcel build $(find frontend -name \\*.html) -d frontend-dist",
+		"build": "parcel build $(find frontend -name \\*.html) --dist-dir frontend-dist",
 		"dev": "cargo watch -i frontend-dist -s 'npm run build'"
 	},
 	"dependencies": {},
 	"devDependencies": {
 		"eslint-config-xo": "^0.37.0",
-		"parcel-bundler": "^1.12.5",
-		"parcel-plugin-svelte": "^4.0.8",
-		"posthtml-include": "^1.7.1",
-		"svelte": "3.31.2",
-		"xo": "0.39.0"
+		"eslint-plugin-promise": "^5.1.1",
```

Comment:
Something broke builds on my machine.

Log: https://bin.wolo.dev/?a9cbc2b45a13584b#GXx7oJtiQds3Za3knfnnjoiTb8oARqLjq2YLHQUKgypQ

Reply:
Hm, not sure why that is. Could you try again with the new version?

Reply:
It works with the new changes, but only after I cleaned package-lock.json and node_modules again.